libs/SalesforceRestAPI/SalesforceRestAPI/Classes/SFRestAPI+Files.m

```diff
@@ -40,41 +40,41 @@
 @implementation SFRestAPI (Files)
 
 - (SFRestRequest *) requestForOwnedFilesList:(NSString *)userId page:(NSUInteger)page {
-    NSString *path = [NSString stringWithFormat:@"/%@/chatter/users/%@/files", self.apiVersion, (userId == nil ? ME : userId)];
```

Comment:
Default is ```v36.0```, so we should be okay. There's no reason for users to downgrade API version on new SDK. Typically internal consumers upgrade to pre-rel.